flake-lang/flake-rust.nix

```
@@ -54,6 +89,7 @@ let
 in
 {
   devShells."dev-${crateName}-rust" = craneLib.devShell {
+    buildInputs = nativeBuildInputs;
```

Any particular reason we don't have the following?
```suggestion
    inherit nativeBuildInputs;
```
It seems to me that we'd want the `nativeBuildInputs` to be the `nativeBuildInputs`?

---

A bit confusingly, but `nativeBuildInputs` are not present in the devshell, so if I run `cargo build`, I won't have access to them (Mac system frameworks in my specific case).